README.md

```diff
@@ -56,35 +65,35 @@ $ vagrant ssh
 vagrant@vagrant:/vagrant$ ./scripts/server
 ```
 
-### Hot Reloading 🔥
-
-Because the frontend uses [Create React App](https://github.com/facebook/create-react-app/), which integrates with webpack, the page will automatically reload if you make changes to the code.
-
-In development, the [Django](https://www.djangoproject.com) app sits behind a [Gunicorn](https://www.gunicorn.org) worker that is passed the [`--reload` flag](https://docs.gunicorn.org/en/stable/settings.html#reload).
-
-### Development Data
+### Development Data 📈
 
-To destroy any existing development database and load fresh fixture data including users, facility lists, facility matches, and facilities run:
+To destroy the existing development database and load fresh fixture data including users, facility lists, facility matches, and facilities, run:
```

Comment:
We may want to consider moving this to the top of the "Development" section and directing people to run `resetdb` before `server`. I set things up from scratch on this branch and the app is pretty broken unless there is some content in the database.